### PR TITLE
This commit adds code to calculate time taken to build a wheel

### PR DESCRIPTION
**Pull request by rd4398**
Changes as per suggestion in https://github.com/python-wheel-build/fromager/pull/192#pullrequestreview-2183301298

### CODE REVIEW

**Review by shubhbapna — COMMENTED** [review]
Looks good! But will wait for @dhellmann approval for merge so as to not block this weeks builds

File: src/fromager/wheels.py
Comment (dhellmann) [inline src/fromager/wheels.py:110]: Does this output go to stdout?

What happens to the `fromager step build` command output with this info log message here? The command is supposed to print the filename it creates in a way that a shell script can grab it and do something with it. Will that still work with this change here?

Comment (rd4398) [reply src/fromager/wheels.py:110]: Yeah, I can see it on terminal and in file `*-prepare-build.log` file
I also ran `test_build_steps.sh` to see if it works.

Is there anything else / another way that I need to test?

Comment (dhellmann) [reply src/fromager/wheels.py:110]: What happens if you try to capture the output of a build command? Something like `wheel=$(fromager step build ...)`. Do you get just the filename, or does it include this new log message, too?

Comment (rd4398) [reply src/fromager/wheels.py:110]: I get just the filename
I ran the following command
```
OUTPUT="$(fromager \
        --log-file "$OUTDIR/build-logs/${dist}-prepare-build.log" \
        --work-dir "$OUTDIR/work-dir" \
        --sdists-repo "$OUTDIR/sdists-repo" \
        --wheels-repo "$OUTDIR/wheels-repo" \
        --wheel-server-url "${WHEEL_SERVER_URL}" \
        step build-wheel "$dist" "$version")"

echo "${OUTPUT}"
```
I get output as

`++ echo /home/rdevasth/fromager/e2e-output/wheels-repo/build/stevedore-5.2.0-py3-none-any.whl`

Comment (dhellmann) [reply src/fromager/wheels.py:110]: Good, that's what I was hoping.

**Review by dhellmann — APPROVED** [review]

**Pull request merged**
